test/neural_networks/test_circuit_qnn.py

```diff
@@ -196,89 +194,37 @@ def _verify_qnn(
-        (True, True, CUSTOM_PASS_MANAGERS, 1, 2),
-        (True, True, CUSTOM_PASS_MANAGERS, 2, 1),
-        (True, True, CUSTOM_PASS_MANAGERS, 2, 2),
-        (True, False, STATEVECTOR, 0, 1),
-        (True, False, STATEVECTOR, 0, 2),
-        (True, False, STATEVECTOR, 1, 1),
-        (True, False, STATEVECTOR, 1, 2),
-        (True, False, STATEVECTOR, 2, 1),
-        (True, False, STATEVECTOR, 2, 2),
-        (True, False, QASM, 0, 1),
-        (True, False, QASM, 0, 2),
-        (True, False, QASM, 1, 1),
-        (True, False, QASM, 1, 2),
-        (True, False, QASM, 2, 1),
-        (True, False, QASM, 2, 2),
-        (True, False, CUSTOM_PASS_MANAGERS, 0, 1),
-        (True, False, CUSTOM_PASS_MANAGERS, 0, 2),
-        (True, False, CUSTOM_PASS_MANAGERS, 1, 1),
-        (True, False, CUSTOM_PASS_MANAGERS, 1, 2),
-        (True, False, CUSTOM_PASS_MANAGERS, 2, 1),
-        (True, False, CUSTOM_PASS_MANAGERS, 2, 2),
-        (False, True, STATEVECTOR, 0, 1),
-        (False, True, STATEVECTOR, 0, 2),
-        (False, True, STATEVECTOR, 1, 1),
-        (False, True, STATEVECTOR, 1, 2),
-        (False, True, STATEVECTOR, 2, 1),
-        (False, True, STATEVECTOR, 2, 2),
-        (False, True, QASM, 0, 1),
-        (False, True, QASM, 0, 2),
-        (False, True, QASM, 1, 1),
-        (False, True, QASM, 1, 2),
-        (False, True, QASM, 2, 1),
-        (False, True, QASM, 2, 2),
-        (False, True, CUSTOM_PASS_MANAGERS, 0, 1),
-        (False, True, CUSTOM_PASS_MANAGERS, 0, 2),
-        (False, True, CUSTOM_PASS_MANAGERS, 1, 1),
-        (False, True, CUSTOM_PASS_MANAGERS, 1, 2),
-        (False, True, CUSTOM_PASS_MANAGERS, 2, 1),
-        (False, True, CUSTOM_PASS_MANAGERS, 2, 2),
-        (False, False, STATEVECTOR, 0, 1),
-        (False, False, STATEVECTOR, 0, 2),
-        (False, False, STATEVECTOR, 1, 1),
-        (False, False, STATEVECTOR, 1, 2),
-        (False, False, STATEVECTOR, 2, 1),
-        (False, False, STATEVECTOR, 2, 2),
-        (False, False, QASM, 0, 1),
-        (False, False, QASM, 0, 2),
-        (False, False, QASM, 1, 1),
-        (False, False, QASM, 1, 2),
-        (False, False, QASM, 2, 1),
-        (False, False, QASM, 2, 2),
-        (False, False, CUSTOM_PASS_MANAGERS, 0, 1),
-        (False, False, CUSTOM_PASS_MANAGERS, 0, 2),
-        (False, False, CUSTOM_PASS_MANAGERS, 1, 1),
-        (False, False, CUSTOM_PASS_MANAGERS, 1, 2),
-        (False, False, CUSTOM_PASS_MANAGERS, 2, 1),
-        (False, False, CUSTOM_PASS_MANAGERS, 2, 2),
-    )
     @unittest.skipIf(not _optionals.HAS_SPARSE, "Sparse not available.")
-    def test_circuit_qnn(self, config):
+    def test_circuit_qnn(self):
         """Circuit QNN Test."""
-        # get configuration
-        sparse, sampling, quantum_instance_type, interpret_id, batch_size = config
-
-        # get QNN
-        qnn = self._get_qnn(sparse, sampling, quantum_instance_type, interpret_id)
+        for sparse in [True, False]:
```

Comment:
I will note that we did things like this before with ddt https://github.com/Qiskit/qiskit-aqua/blob/1f2c316c3a1aca1296f45241d14ad8ae5fbe2027/test/aqua/test_deutsch_jozsa.py#L24-L35

Reply:
Perfect, this is what I was looking for. Updated the tests.